### PR TITLE
MINOR: Modify unnecessary access specifiers

### PR DESCRIPTION
**Pull request by highluck**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by highluck** [comment]
@chia7712 
May I ask for a review of this PR?

**Comment by highluck** [comment]
Ping
@chia7712
May I ask for a review of this PR?

File: streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java [hunk 1]
```diff
@@ -104,7 +104,7 @@
     private final RocksDBMetricsRecorder metricsRecorder;
 
     // visible for testing
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java]: Is this comment still valid?

**Review by chia7712 — COMMENTED** [review]
@highluck thanks for your patch. one comment is left.

File: streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java (hunk 1 above)
Comment (highluck) [reply streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java]: Thanks for the comment.
That part doesn't seem necessary.
And I think you can clear that field.

**Comment by highluck** [comment]
@chia7712
Thank you for the comment Code was modified

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java [hunk 2]
```diff
@@ -96,7 +96,7 @@ public boolean putIfDifferentValues(final K key,
     public static class RawAndDeserializedValue<ValueType> {
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java]: It seems this class is used by testing only. Could you change it from public to package-private?

**Review by chia7712 — COMMENTED** [review]
@highluck thanks for your patch. one minor comment is left.

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java (hunk 2 above)
Comment (highluck) [reply streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java]: @chia7712 The code has been modified!

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java [hunk 3]
```diff
@@ -58,7 +58,7 @@
     }
 
 
-    public RawAndDeserializedValue<V> getWithBinary(final K key) {
+    RawAndDeserializedValue<V> getWithBinary(final K key) {
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java]: This change may be overkill as this is not an inner class.

File: streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java [hunk 4]
```diff
@@ -70,9 +70,9 @@
         }
     }
 
-    public boolean putIfDifferentValues(final K key,
-                                        final ValueAndTimestamp<V> newValue,
-                                        final byte[] oldSerializedValue) {
+    boolean putIfDifferentValues(final K key,
```
Comment (chia7712) [inline streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java]: ditto

**Review by chia7712 — COMMENTED** [review]
@highluck thanks for updating code. a couple of comments. please take a look.

**Comment by highluck** [comment]
@chia7712 
Thank you for the comment Code was modified

**Pull request merged**